### ISSUE TITLE
Switch from pip to pip3

### ISSUE THREAD

**Issue by tomjn** [Varying-Vagrant-Vagrants/VVV#2408]
We should use pip3 to install `shyaml` instead of `pip`. This will help with compatibility as `pip` is not available on Ubuntu 20, but `pip3` is.

**Comment by evertiro** [comment]
I'm re-opening this. On a clean install, this results in a failure due to it trying to force a package removal when there's nothing to remove.

**Comment by evertiro** [comment]
```
default: Reading package lists...
    default: Building dependency tree...
    default: 
    default: Reading state information...
    default: E: Unable to locate package python-pip
    default:  * Removing apt-get packages returned a failure code, cleaning up apt caches then exiting
    default:  ! Main packages removal failed, halting provision
    default:  ! The 'main' provisioner ran into problems, the full log is available at '/var/log/provisioners/2021.03.26_11-44-57/provisioner-main.log'. It completed in 11 seconds.
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by tomjn** [comment]
so it needs to test if `python-pip` is installed before removing
